doc/extending.md

```diff
@@ -41,24 +41,43 @@ directory kept next to your traject config files:
 ~~~
 
 
-The `my_macros.rb` file might contain a simple [macro](./macros.md)
-in a module called `MyMacros`.
+The `my_macros.rb` file might contain a simple [macro](./indexing_rules.md#back-to-macros)
+in a module called `MyMacros` and a method called `my_some_macro`. Let's say it looks like this:
+
+```rb
+module MyMacros
+  def my_some_macro
+    lambda do |_record, accumulator, context|
+      oclc_number = context.output_hash&.dig('oclc_number_ssim')&.first
+      accumulator << "#{oclc_number} my some text"
+    end
+  end
+end
+```
 
 The `my_utility.rb` file might contain, say, a module of utility
 methods, `MyUtility.some_utility`, etc.
 
+```rb
+module MyUtility
+  def some_utility(record)
+    'some text' unless record.fields('505').empty?
+  end
```

Comment:
For this to work as in the examples, I think it has to be `def self.some_utility`, no?  So it can be called like `MyUtility.some_utility`?